easybuild/easyconfigs/p/Perl/Perl-5.26.1-foss-2018a-bare.eb

```diff
@@ -0,0 +1,18 @@
+name = 'Perl'
+version = '5.26.1'
+versionsuffix = '-bare'
+
+homepage = 'http://www.perl.org/'
```

Comment:
@RvDijk any reason why you've changed the URL from `https`?

Reply:
@verdurin probably from an old template, I'll change it to use https